src/core/drive/history.ts

```diff
@@ -1,9 +1,8 @@
-import { Location } from "../location"
 import { Position } from "../types"
 import { nextMicrotask, uuid } from "../../util"
 
 export interface HistoryDelegate {
-  historyPoppedToLocationWithRestorationIdentifier(location: Location, restorationIdentifier: string): void
+  historyPoppedToLocationWithRestorationIdentifier(location: URL, restorationIdentifier: string): void
```

Comment:
There are many methods that reference `Location` in their name that now accept `URL` instances with arguments named `location`.

A global search and replace would be a simple way to rename them, but I'm not sure which are part of the public API. Are there backward compatibility concerns with method names are positional argument names?

Reply:
Thanks for noting. I'll make a pass through and clean these up.

Reply:
Thank you @sstephenson ! I'm all finished on my end if you're ready to take over.

Reply:
Decided not to rename them because it'd be too invasive (the adapter interface would change) and it's fine for the term "location" to remain part of the Turbo Drive domain language.